### PR TITLE
Discuss issued and presented forms of JPTs

### LINKED ISSUE #96: Evaluate promoting parts of section 7 of JPT to the JWP spec
In evaluating rules described for JPT, it makes sense to describe these in the JWP spec for all applications.

### PR DESCRIPTION
**Pull request by selfissued**
Fixes #96

### CODE REVIEW

**Comment by selfissued** [comment]
Fixes #99

File: draft-ietf-jose-json-proof-token.md [hunk 1]
```diff
@@ -128,25 +128,32 @@ The following is an example JWP Protected Header that includes a `cid`:
...
+Each claim in the issued form of the JPT results in one of three things in the presented form of the JPT:
+1. A disclosed JSON value.
+1. An indicator that the value was not disclosed.
+1. An algorithm-specific proof method.
 
 ## Disclosed
 
-Always an octet string of valid JSON text.
+A disclosed payload is represented as a base64url-encoded UTF-encoded octet string
+representing a valid JSON value.
```
Comment (dwaite) [inline draft-ietf-jose-json-proof-token.md:141]:
```suggestion
A disclosed payload is represented as a UTF-encoded octet string representing a valid JSON value.
```

File: draft-ietf-jose-json-proof-token.md [hunk 2]
```diff
@@ -128,25 +128,32 @@ The following is an example JWP Protected Header that includes a `cid`:
...
+
+## Undisclosed
+
+The placeholder indicating that a payload was not disclosed is represented as
+a `null` value for the JWP JSON Serialization
+or the empty string value for the JWP Compact Serialization.
```
Comment (dwaite) [inline draft-ietf-jose-json-proof-token.md:147]: Recommend we leave this to the core JWP spec. I'm specifically trying to consolidate special cases like this under the serialization rules.

File: draft-ietf-jose-json-proof-token.md (hunk 1 above)
Comment (dwaite) [reply draft-ietf-jose-json-proof-token.md:141]: payloads are binary until serialized

**Review by dwaite — CHANGES_REQUESTED** [review]

File: draft-ietf-jose-json-proof-token.md (hunk 2 above)
Comment (selfissued) [reply draft-ietf-jose-json-proof-token.md]: I updated the text to reference the representation in JWP rather than repeating it.

**Review by dwaite — APPROVED** [review]

**Pull request merged**